### PR TITLE
Sort constants in alphabetical order

### PR DESCRIPTION
**Pull request by GiBg1aN**
I found this trivial abandoned PR, so did it from scratch. (Addressed to #622)

### CODE REVIEW

**Comment by php-coder** [comment]
Hi,
>
>
Thank you for this PR! I’m on holidays now and can’t review or merge it
from my phone. I’ll look at it in 3 days.


-- 
Slava Semushin

File: src/main/java/ru/mystamps/web/Url.java [hunk 1]
```diff
@@ -128,33 +128,34 @@ private Url() {
 		boolean serveContentFromSingleHost = !production;
 		
 		// Not all URLs are listed here but only those that are being used on views
+		// Constant sorted in ascending order
```
Comment (php-coder) [inline src/main/java/ru/mystamps/web/Url.java]: Could you modify it to "Constants sorted in an ascending order." ?

**Comment by GiBg1aN** [comment]
Yeah, you are right, I just fixed them.

**Comment by GiBg1aN** [comment]
Done, check it out

**Comment by php-coder** [comment]
Merged in 78e9e94279e54bc94e53187d9e3f84c99175b773 commit.

Thanks!

**Comment by php-coder** [comment]
@GiBg1aN Let me know if you need more tasks like this one.

**Pull request closed without merging**